app/routes/notes/$noteId.tsx

```diff
@@ -3,8 +3,7 @@ import { json, redirect } from "@remix-run/node";
 import { Form, useCatch, useLoaderData } from "@remix-run/react";
 import invariant from "tiny-invariant";
 
-import { deleteNote } from "~/models/note.server";
-import { getNote } from "~/models/note.server";
+import { deleteNote, getNote } from "~/models/note.server";
```

Comment:
This has nothing to do with enabling the rule, ESLint was just warning for this, so I fixed it